mava/systems/ff_ippo_rware.py

```diff
@@ -558,9 +559,11 @@ def run_experiment(_run: run.Run, _config: Dict, _log: SacredLogger) -> None:
         jax.block_until_ready(evaluator_output)
 
         # Log the results of the evaluation.
+        elapsed_time = time.time() - start_time
+        evaluator_output.episodes_info["steps_per_second"] = steps_per_rollout / elapsed_time
         episode_return = log(
             metrics=evaluator_output,
-            t_env=timesteps_per_training * (i + 1),
+            t_env=steps_per_rollout * (i + 1),
         )
         if config["absolute_metric"] and max_episode_return <= episode_return:
```

Comment:
We need to add `evaluator_output.episodes_info["steps_per_second"] = steps_per_rollout / elapsed_time` to the absolute metric else it will cause an error of not finding steps_per_seconds within it

Reply:
Great spot!